charts/presidio/values.yaml

```diff
@@ -10,6 +10,11 @@ redis:
     #db: 0
     #ssl: false
 
+# Update url with the internal redis url, phase 2
+redis_internal:
+    nameOverride: persistent-redis
+    url: redis-master.presidio-system.svc.cluster.local:6379
```

Comment:
Is this Redis deployed by the Chart or expected to already exist in the cluster? Probably me being silly but I can't find any reference to it in the Chart information.

Reply:
This redis is **CURRENTLY**  deployed explicitly before Presidio is deployed.
in [this PR](https://github.com/Microsoft/presidio/pull/106), I am installing Redis internally with Presidio.